source/utilities.tex

```diff
@@ -16511,8 +16511,8 @@
 namespace std {
   template<class Context>
   class basic_format_arg<Context>::handle {
-    const void* ptr_;                                           // \expos
-    void (*format_)(basic_format_parse_context<char_type>&,
+    const void* @\exposid{ptr_}@;                                           // \expos
+    void (*@\exposid{format_}@)(basic_format_parse_context<char_type>&,
```

Comment:
```suggestion
    void (*@\exposid{format_}@)(basic_format_parse_context<\exposid{char-type}>&,
```

Reply:
More missed cases below in the definition of `handle` and its members.

Reply:
Lines 16523, 16558, and 16570 (at least).

Reply:
I believe I have now caught all missing usage of \exposid, but I thought that before too!